File: PoliticsNg/src/app/_models/Activity.ts
import { User } from './User';

export interface ActivityLike{
    likes: number[];
    dislikes: number[];
}

export interface ActivityComment {
    id: number,
    content: string,
    side: string,
    likes: number[],
    user: User,
}
export interface Activity {
    id: number;
    title: string;
    description: string;
    comments: ActivityComment[];
    likes: ActivityLike;
}

File: PoliticsNet/Models/Activity.cs
using System;
using System.Collections.Generic;

namespace PoliticsNet.Models
{
    public class Activity
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public Boolean Visible { get; set; }
        public virtual ICollection<ActivityComment> ActivityComments { get; set; }
        public virtual ICollection<ActivityLike> ActivityLikes { get; set; }

        public Activity(){
            ActivityComments = new List<ActivityComment>();
            ActivityLikes = new List<ActivityLike>();
        }
    }
}